### DISCUSSION TITLE
F10 / F11 / F12 Command line Options

### DISCUSSION THREAD

**Discussion by Ohrwerk** [AliveTeam/alive_reversing#1607] [Q&A] [answered]
Does anyone know if there are command line options for Relive.
I would like to start the game directly in fullscreen mode with smoothing enabled.
Thank you

**Comment by mouzedrift** [comment, marked as answer]
there are currently no command line options for these things but you can configure the `abe2.ini` config file and set `filter_screen` to `true` but there is no option for fullscreen atm. the `abe2.ini` file should be in the game directory but you might need to go to the ingame config screen and then quit the game to create the config file in case it doesn't exist already.

**Comment by Ohrwerk** [comment]
Thank you
